Assignment3.txt

```diff
@@ -0,0 +1,11 @@
+Mariposa Lee 
+
+1a. o(n) = n. It will run until i is less than n
+b. o(n) = n+n^2. First, it will run n times, then n times for every n times (the for loops). 
```

Comment:
Close! since it's big O notation, you don't have to care about the lower terms (such as n), so this just becomes O(n^2).